### PR TITLE
Authentication and cart management

### PR DESCRIPTION
**Pull request by samyakshah3008**
Live link: https://jovial-roentgen-c7743d.netlify.app/ 

FEAT: Two new screens are now added on tripleS Store 

Feedbacks are welcomed!

### CODE REVIEW

**Review by pritam-kr — APPROVED** [review]

**Comment by pritam-kr** [comment]
Good to go

**Review by PeeyushPrashant — APPROVED** [review]
looks good, just try to raise PR for each component separately, as it gets easier for others to review

**Review by j24m — APPROVED** [review]

File: styles/authentication/login.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <nav>
+        <div class="nav-container">
+            <a href="/index.html"><div class="nav-logo">TripleS Store</div></a>
+         <div class="input-container">
+         <input type="text" class="nav-input" placeholder="Search">
+         <span class="fas fa-search search-btn"></span>
+     </div>
+            
+     <a href="/styles/authentication/signup.html"><button class="signup-btn">Sign up</button></a>
+
```
Comment (Meghana-sk) [inline styles/authentication/login.html:22]: empty lines can be removed

File: styles/authentication/signup.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <nav>
+        <div class="nav-container">
+         <a href="/index.html"><div class="nav-logo">TripleS Store</div></a>
+         <div class="input-container">
+         <input type="text" class="nav-input" placeholder="Search">
+         <span class="fas fa-search search-btn"></span>
+     </div>
+            
+     <a href="/styles/authentication/login.html"><button class="signup-btn">Log in</button></a>
+
```
Comment (Meghana-sk) [inline styles/authentication/signup.html:22]: empty lines can be removed

File: styles/cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+.pay-info{
+    display: flex;
+    justify-content: space-between;
+}
+
+.total-price-label{
+    max-width: 350px;
+    flex-basis: 140px;
+
```
Comment (Meghana-sk) [inline styles/cart/cart.css:39]: remove empty lines

File: styles/cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.border-bottom{
+    border-bottom: 1px solid var(--primary);
+
```
Comment (Meghana-sk) [inline styles/cart/cart.css:44]: remove empty lines

File: styles/cart/cart.html [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        <i class="fas fa-star star selected-star"></i>
+        <i class="fas fa-star star selected-star"></i>
+        <i class="fas fa-star star selected-star"></i>
+        <i class="fas fa-star star selected-star"></i>
+        <i class="far fa-star star selected-star"></i>
+        <div class="product-price">$60.00</div>
+        <div class="container3">
+            <button class="button2 full-width">Remove from Cart</button>
+        </div> 
+
```
Comment (Meghana-sk) [inline styles/cart/cart.html:62]: I would suggest you to remove all empty lines which will increase Line of code simply

**Review by Meghana-sk — APPROVED** [review]
Great job Samyak :). Have few suggestions:

1. Input search filter - text is overlapping with search icon
image

2. All empty lines can be removed throught files

**Pull request merged**